src/pages/ContactPage.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import BaseLayout from '../layout/BaseLayout';
+
+import '../styles/ContactPage.scss';
+
+type Props = {
+  location: {
+    state: {
+      accession?: string;
+    }
+  }
+} & RouteComponentProps;
+
+const ContactPage: React.FC<Props> = ({
+  location: { state: { accession = '' } },
+}) => {
+  const [subject, setSubject] = useState('');
+  const [agree, setAgree] = useState(false);
+
+  useEffect(() => {
+    if (accession.length > 0) {
+      setSubject(`Covid-19 UniProtKB/Swiss-Prot ${accession} entry update request`);
+    }
+  }, [accession]);
+
+  return (
+    <BaseLayout>
+      <form action="https://www.uniprot.org/feedback" method="POST">
+        <input type="hidden" name="entry" value={accession} />
+        <div className="contact-page-form">
+          <h3>Contact</h3>
+          <div className="contact-page-form__row">
+            <span>
+              Name
+            </span>
+            <input
+              id="name"
+              type="text"
+              name="name"
+            />
+          </div>
+          <div className="contact-page-form__row">
+            <span>
+              Email
+            </span>
+            <input
+              id="email"
+              type="text"
+              name="email"
+            />
+          </div>
+          <div className="contact-page-form__row">
+            <span>
+              Subject
+            </span>
+            <input
+              id="subject"
+              type="text"
+              name="subject"
+              value={subject}
+              onChange={e => setSubject(e.target.value)}
+            />
+          </div>
+          <div className="contact-page-form__row">
+            <span>
+              Message
+            </span>
+            <textarea
+              id="message"
+              name="message"
+            />
+          </div>
+          <div className="contact-page-form__row">
+            <span />
+            <input
+              name="privacy_ok"
+              type="checkbox"
+              checked={agree}
+              onChange={e => setAgree(e.target.checked)}
+            />
+            &nbsp;
+            I agree to the processing of my data for the purposes described in this
+            &nbsp;
+            <a href="https://www.uniprot.org/help/privacy">privacy notice</a>.
+          </div>
+          <div className="contact-page-form__row">
+            <span />
+            {agree
```

Comment:
I think you can just use the conditional on the attribute `disabled=""`

Reply:
I remember `disabled="false"` wouldn't work in some browsers, but now I have switched to a conditional.